personal-portfolio/css/style.css

```diff
@@ -750,14 +750,17 @@ button.btn-1:hover {
 
```

Comment:
Pop-up still does not follow the design on both desktop and mobile. `See live` and `See Source` buttons should be on the right side of the project title. The mobile view does not show these buttons. Kindly make the necessary changes while following the design exactly
![btns](https://user-images.githubusercontent.com/35504149/136542339-87cc395d-d691-4b13-b1af-97e0a0d316a5.png)
.